src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -67,27 +65,38 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
   public void addTermRating(HttpServletRequest request) throws IOException {
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-
-    StringBuilder jb = new StringBuilder();
+    StringBuilder stringBuilder = new StringBuilder();
     String line = null;
     try {
       BufferedReader reader = request.getReader();
-      while ((line = reader.readLine()) != null) jb.append(line);
+      while ((line = reader.readLine()) != null) stringBuilder.append(line);
```

Comment:
Also FYI I think Google's style guide prefers to always have {} when using while/if/etc statements. Just for added clarity.